hack/godep-restore.sh

```diff
@@ -56,8 +56,7 @@ preload-remote "github.com/openshift" "origin" "github.com/openshift" "origin" #
 preload-remote "k8s.io" "kubernetes" "github.com/openshift" "kubernetes"
 preload-remote "github.com/docker" "distribution" "github.com/openshift" "docker-distribution"
 preload-remote "github.com/skynetservices" "skydns" "github.com/openshift" "skynetservices-skydns"
-preload-remote "github.com/coreos" "etcd" "github.com/openshift" "coreos-etcd"
```

Comment:
@smarterclayton why did we have this?
@soltysh do we no longer have any upstream commits?

Reply:
I think we had those some time ago, but I've double checked the last bump and it was using official version (no carry commits). That's why I've decided to remove it from here. We'll bring it back when needed.